rollup.config.js

```diff
@@ -12,6 +12,18 @@ import css from "rollup-plugin-css-only";
 
 const production = !process.env.ROLLUP_WATCH;
 
+function extractLastUpdatedTime(str) {
+  const json = JSON.parse(str);
+  const lastUpdatedDate = new Date();
+  lastUpdatedDate.setTime(Date.parse(json.lastUpdate));
+
+  const month = lastUpdatedDate.toLocaleString("default", { month: "long" });
+  const year = lastUpdatedDate.toLocaleString("default", { year: "numeric" });
+  const day = lastUpdatedDate.toLocaleString("default", { day: "numeric" });
+
+  return `${month} ${day}, ${year}`;
```

Comment:
I would suggest using [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) here instead. Aside from that, LGTM!

```suggestion
  return lastUpdatedDate..toLocaleDateString('en-US', { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
```